demos/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "homepage": "https://github.hubspot.com/calling-extensions-sdk",
+  "scripts": {
+    "ci:js": "cd demo-minimal-js && npm ci",
+    "ci:react": "cd demo-react-ts && npm ci",
+    "build:js": "cd demo-minimal-js && npm run build",
+    "build:react": "cd demo-react-ts && npm run build",
+    "test:react": "cd demo-react-ts && npm run test",
+    "ci": "npm run ci:js && npm run ci:react",
+    "build": "npm run build:js && npm run build:react",
+    "test": "npm run test:react",
+    "predeploy": "npm run ci && npm run build && cp ./src/index.html build/index.html && cp -a demo-minimal-js/bin/. build && cp -a demo-react-ts/dist/. build"
```

Comment:
We have separate steps for `npm install` and `npm run build` in https://github.com/HubSpot/calling-extensions-sdk/blob/master/demo/package.json as well. Which npm command are you thinking of using to combine these 2 steps?

Reply:
Considerations:
- Being consistent with how the current demo app is installed and built.
- You might not want to install deps everytime you want to build.
Huddle Suggestion:
dev && ci commands separate
```
cd demo-min-js
"build"
"build:gha": cd && npm ci && build
cd demo-react-ts
"build"
"build:gha": cd && npm ci && build
demos
prebuild:gha:
build:gha
postbuild:gha 
```

Reply:
Thank you! I've taken these suggestions into consideration and renamed the scripts. I've added a description to the package.json and 'm thinking that we can keep the commands in the `demos` folder simple since there are no other files that the `build` script could possibly refer to